package.json

```diff
@@ -58,5 +75,20 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.12.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.+(js|jsx)": [
+      "yarn lint:fix",
+      "git add"
```

Comment:
Just to confirm, we want this to be automatic, right? In other projects I usually get notified, but it doesn't fix and add changes automatically; but I understand that this "more comfortable" for devs.

Reply:
Indeed, we can open a discussion for that, but, it's using same config as template.

Reply:
Agree, same as above.